tests/tests/test_monitor_client.py

```diff
@@ -274,7 +274,7 @@ def test_monitorclient_alert_email(self, monitor_commercial_setup_no_client):
         """Tests the monitor client email alerting"""
         mailbox_path = "/var/spool/mail/local"
         wait_for_alert_interval_s = 8
-        expected_from = "noreply@mender.io"
+        expected_from = "no-reply@hosted.mender.io"
```

Comment:
`noreply@mender.io"` was something that was agreed with Eystein, I know these are only tests -- but do we have a good reason not to keep noreply@mender.io here?

Reply:
Because of SPF/DMARC/DKIM settings, we cannot send emails for `@mender.io` from a non-Google SMTP server. That's why we have to move to `@hosted.mender.io`. We can keep whatever we want in the test, but I tried to keep it as close as possible to the reality.

Reply:
exactly my point to keep it close to reality :) if we now cannot send from @medner.io, then it is fine. thanks. I will let Eystein know.